cookbook/spec_add_ethereum.json

```diff
@@ -170,16 +170,16 @@
                         "name": "eth_hashrate",
                         "block_parsing": {
                             "parser_arg": [
-                                ""
+                                "latest"
                             ],
-                            "parser_func": "EMPTY"
+                            "parser_func": "DEFAULT"
                         },
                         "compute_units": "1",
                         "enabled": true,
                         "api_interfaces": [
                             {
                                 "category": {
-                                    "deterministic": false,
+                                    "deterministic": true,
```

Comment:
this is not deterministic
https://docs.infura.io/infura/networks/ethereum/json-rpc-methods/eth_hashrate
can change per node

Reply:
done